sdk/include/psdk/winnls.h

```diff
@@ -666,7 +666,7 @@ typedef struct _numberfmtW {
 	LPWSTR lpThousandSep;
 	UINT NegativeOrder;
 } NUMBERFMTW,*LPNUMBERFMTW;
-#if (WINVER >= 0x0600)
+#if 1//(WINVER >= 0x0600)
```

Comment:
Why these `1`s? Do you mean `#if (WINVER >= 0x0600) || defined(__REACTOS__)` or similar?